spec/factories/games.rb

```diff
@@ -2,7 +2,7 @@
   factory :game do
     sequence(:title) { |n|"game #{n}" }
     age_rating { 1 }
-    agg_rating {rand(1..100)}
+    agg_rating {rand(11..90)}
```

Comment:
this was to get around some errors with sad path and there still might be some. 
If you expect a rating (44) to not be on the page the game title is `game 44` then it will fault out. It doesn't brake consistently but it can be annoying